### DISCUSSION TITLE
Welcome to Manifold Discussions!

### DISCUSSION THREAD

**Discussion by zdavis** [ManifoldScholar/manifold#2902] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jojokarlin** [comment]
Hi Manifold! 👋 
I'm Jojo, recent GC English alum and former Manifold Graduate Fellow, now working in Digital Scholarship Services at NYU Libraries. I'm really excited to see how Manifold moves forward for OER with the upcoming grants. I'd love to connect with OER users and library publishers across the growing Manifold community.
I look forward to connecting with more people on Github.
-Jojo
